_assets/css/_home.scss

```diff
@@ -180,12 +180,27 @@
 
   .banner.bountysource {
     margin-top: 0;
-    .bountysource-logo {
-      height: 25px;
+    .flag-logos {
+      display: inline;
```

Comment:
I'd prefer `display: flex`, it's easier to handle.

Reply:
We are fine with inline. As long as it does the jobs. The scope is pretty narrow of it.